src/components/ShoppingCartItem.js

```diff
@@ -1,6 +1,7 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
```

Comment:
I think you had some plans to `useState()` and `useEffect)()` - don't forget to remove the imports if you don't end up using them '' '